package.json

```diff
@@ -1,14 +1,25 @@
 {
-  "name": "chorus",
-  "private": true,
-  "type": "module",
-  "version": "1.22.2",
-  "scripts": {
-    "build": "rollup -c",
-    "watch": "rollup -c -w"
-  },
-  "devDependencies": {
-    "rollup": "3.28.1",
-    "rollup-plugin-chrome-extension": "3.6.12"
-  }
+    "name": "chorus",
+    "private": true,
+    "type": "module",
+    "version": "1.22.2",
+    "scripts": {
+        "build": "rollup -c",
+        "watch": "rollup -c -w",
+        "prepare": "husky"
```

Comment:
I just remembered from a different project that we don't want husky to run in the CI at will error in our Github action. We don't want anymore release issues. 😄 I recall using the solution [here](https://typicode.github.io/husky/how-to.html#ci-server-and-docker) where an `install.mjs` file was included?